ova/generate_ova.sh

```diff
@@ -150,13 +150,13 @@ main() {
             shift 2
         ;;
 
-        "-e" | "--elasticsearch")
+        "f" | "--filebeat")
```

Comment:
```suggestion
        "-f" | "--filebeat")
```

Reply:
Done in  528feec208a367ba616fb87ea4237096d0681494